app/controllers/collections_controller.rb

```diff
@@ -100,6 +101,27 @@ def history
 
   private
 
+  # @return [ActiveRecord::Relation] the works to display for the collection, filtered by search term if applicable
+  def collection_works_result
+    collection_works_result = @collection.works
```

Comment:
Slight suggestion:
```
collection_works_result = @collection.works

return collection_work_result if @search_term.blank?

collection_works_result.joins(
 ...
```

Reply:
ended up tightening it up a bit more than that, and got rid of the `collection_works_result` temp variable, since it doesn't really get built up anymore after this refactoring.  if there's more conditional filtering or sorting to be added later, someone can add a temp var for building the `ActiveRecord::Relation` at that point.  (i'd put in the temp var in when the query building was directly in the controller method, primarily because i thought that made it slightly more readable there -- now that there's a separate helper method, the temp var feels like it adds less to me)